src/Microsoft.TestPlatform.Extensions.CoverageLogger/CoverageLogger.cs

```diff
@@ -0,0 +1,247 @@
+                        ConsoleOutput.Instance.Information(resultFile);
+
+                        this.ProcessCoverageXml(resultFile);
+                    }
+                    else
+                    {
+                        EqtTrace.Error("CodeCoverage.exe could not process file within 10 secs");
+                        if (!this.vanguardProcess.HasExited)
+                        {
+                            this.vanguardProcess.Kill();
+                        }
+                    }
+
+                    this.vanguardProcess.Dispose();
+                }
+            }
+        }
+
+        private void ProcessCoverageXml(string coverageXml)
+        {
+            using (XmlReader reader = XmlReader.Create(coverageXml, ReaderSettings))
+            {
+                var settingsDocument = new XmlDocument();
+                settingsDocument.Load(reader);
+
+                var modules = settingsDocument.GetElementsByTagName("module");
+
+                if (modules.Count > 0)
+                {
+                    ConsoleOutput.Instance.Information("\n" + string.Format(CultureInfo.CurrentCulture, CodeCoverageLoggerResources.CodeCoverageHeader));
+                    ConsoleOutput.Instance.Information("----------------");
+
+                    var header = string.Concat(
+                        string.Format(CultureInfo.CurrentCulture, CodeCoverageLoggerResources.ModuleName),
+                        "\t\t\t",
+                        string.Format(CultureInfo.CurrentCulture, CodeCoverageLoggerResources.NotBlocksCovered),
+                        "\t\t",
+                        string.Format(CultureInfo.CurrentCulture, CodeCoverageLoggerResources.PercNotBlocksCovered),
+                        "\t\t",
+                        string.Format(CultureInfo.CurrentCulture, CodeCoverageLoggerResources.BlocksCovered),
+                        "\t\t",
+                        string.Format(CultureInfo.CurrentCulture, CodeCoverageLoggerResources.PercBlocksCovered),
+                        "\n");
+
+                    ConsoleOutput.Instance.Information(header);
+                }
+
+                foreach (XmlNode moudle in modules)
+                {
+                    ConsoleOutput.Instance.Information(moudle.Attributes["name"].Value + "\t\t" +
+                                                       moudle.Attributes["blocks_not_covered"].Value + "\t\t\t\t\t" +
+                                                       (100 - float.Parse(moudle.Attributes["block_coverage"].Value)) + "\t\t\t\t" +
+                                                       moudle.Attributes["blocks_covered"].Value + "\t\t\t\t" +
+                                                       moudle.Attributes["block_coverage"].Value);
+                }
+            }
+        }
+
+        private void AnalyzeCoverageFile(string arguments)
+        {
+            try
+            {
+                string vsInstallPath = this.GetVSInstallPath();
```

Comment:
I guess CodeCoverage.exe is shipped only with Visual Studio Enterprise, and therefore, this might not work.